apps/catalogue/src/main.js

```diff
@@ -64,7 +65,7 @@ const router = new VueRouter({
     },
     {
       name: "Networks",
-      path: "/networks",
+      path: "/networksold",
```

Comment:
wha is this ?

Reply:
renamed back. And called the new parts 'networks-new'